pyaedt/emit_core/results/revision.py

```diff
@@ -372,34 +372,34 @@ def notes(self, notes):
         self.emit_project._emit_api.save_project()
 
     @property
-    def max_n_to_1_instances(self):
+    def n_to_1_limit(self):
         """
         The maximum number of instances per band combination allowed to run for N to 1.
```

Comment:
```suggestion
       Maximum number of instances per band combination allowed to run for N to 1.
```
We drop the article (The, An, A) at the beginning of the brief description..